aboutlibraries-core/src/androidMain/kotlin/com/mikepenz/aboutlibraries/util/AndroidParser.kt

```diff
@@ -4,9 +4,12 @@ import android.util.Log
 import com.mikepenz.aboutlibraries.entity.*
 import org.json.JSONObject
 
-actual fun parseData(json: String): Result {
+actual fun parseData(json: String, recoverable: Boolean): Result {
     try {
         val metaData = JSONObject(json)
+        val firstItem: JSONObject = metaData.getJSONArray("libraries")[0] as JSONObject
+        firstItem.remove("name")
+        firstItem.remove("developers")
```

Comment:
I believe this was accidentally left in for testing purposes, please remove

Reply:
Removed!